TranslationAssistant.Business/DocumentTranslationManager.cs

```diff
@@ -606,8 +654,14 @@ private static void ProcessPowerPointDocument(string outputDocumentFullName,stri
                         }
                     }
 
-                    ReplaceTextsWithTranslation(texts, sourceLanguage, targetLanguage);
-                    ReplaceTextsWithTranslation(notes, sourceLanguage, targetLanguage);
+                    int[] textCounts = ReplaceTextsWithTranslation(texts, sourceLanguage, targetLanguage);
+                    int[] noteCounts = ReplaceTextsWithTranslation(notes, sourceLanguage, targetLanguage);
```

Comment:
After seeing these usages, it might be better to have a separate CharCounts class in addition. Since this seems more like an intermediate calculation. Could have an Add method too.